test/Benchmarks/OrleansBenchmarkGrainInterfaces/MapReduce/DataflowGrainsInterfaces.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Orleans;
+
+namespace OrleansGrainInterfaces.MapReduce
+{
+    public interface IDataflowGrain : IGrainWithGuidKey
+    {
+        Task Complete();
+
+        Task Fault();
+
+        Task Completion();
+    }
+
+    public interface ITargetGrain<in TInput> : IDataflowGrain
```

Comment:
Interface name is very generic.  Suspect this will lead to naming conflicts in future.  Especially if more dataflow related test scenarios are added.  Similar concern for other concrete grain types in this test harness.